File: task-templates/default/public/with-grader/py/grader.py
import sys
import __TPARAM_SHORT_NAME__

def main():
    n = int(input())
    sys.stdin.close()

    res = __TPARAM_SHORT_NAME__.__TPARAM_GRADER_FUNCTION_NAME__(n)

    print(res)
    sys.stdout.close()

if __name__ == '__main__':
    main()

File: task-templates/default/public/with-grader/cpp/grader.cpp
#include "__TPARAM_SHORT_NAME__.h"
#include <cassert>
#include <cstdio>

int main() {
	int n;
	assert(1 == scanf("%d", &n));
	fclose(stdin);

	int res = __TPARAM_GRADER_FUNCTION_NAME__(n);

	printf("%d\n", res);
	fclose(stdout);
	return 0;
}
